### PR TITLE
Add ElkM1 debugging section to docs

### PR DESCRIPTION
**Pull request by gwww**
## Proposed change

Add debugging section to documentation.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/elkm1.markdown [hunk 1]
```diff
@@ -553,3 +553,15 @@ Speak a word. The list of words is defined in the ElkM1 ASCII Protocol documenta
 | ---------------------- | -------- | ----------- |
 | `word` | no | Word to speak.
 | `prefix` | yes | Prefix to identify panel when multiple panels configured.
+
+## Debugging
```
Comment (bdraco) [inline source/_integrations/elkm1.markdown:557]: Maybe link to https://www.home-assistant.io/docs/configuration/troubleshooting/#enabling-debug-logging and only include the manual config if they need it before startup happens?

Comment (gwww) [reply source/_integrations/elkm1.markdown:557]: Thanks. I was aware of the UI for debug. Does it persist over restarts?

Comment (frenck) [reply source/_integrations/elkm1.markdown:557]: No, it does not persist over restart.

File: source/_integrations/elkm1.markdown [hunk 2]
```diff
@@ -553,3 +553,18 @@ Speak a word. The list of words is defined in the ElkM1 ASCII Protocol documenta
...
+Debug logs are often required to solve an issue. Follow the instructions on [Enabling debug logging](https://www.home-assistant.io/docs/configuration/troubleshooting/#enabling-debug-logging).
```
Comment (frenck) [inline source/_integrations/elkm1.markdown]:
```suggestion
Debug logs are often required to solve an issue. Follow the instructions on [Enabling debug logging](/docs/configuration/troubleshooting/#enabling-debug-logging).
```

File: source/_integrations/elkm1.markdown [hunk 3]
```diff
@@ -553,3 +553,18 @@ Speak a word. The list of words is defined in the ElkM1 ASCII Protocol documenta
...
+
+Sometimes, for example, a problem can occur while starting Home Assistant. In this case follow these instructions.
+Add the following to your `configuration.yaml` file in your Home Assistant `config` directory:
```
Comment (frenck) [inline source/_integrations/elkm1.markdown]:
```suggestion
Sometimes, for example, a problem can occur while starting Home Assistant. In this case, follow these instructions.
Add the following to your `configuration.yaml` file in your Home Assistant `config` directory:
```

**Review by frenck — APPROVED** [review]

**Comment by gwww** [comment]
Thanks!

**Comment by bdraco** [comment]
Thanks. Sorry this one fell off my radar

**Pull request merged**